.circleci/config.yml

```diff
@@ -709,11 +725,11 @@ jobs:
             ps axww | grep watcher_info
             ps axww | grep child_chain
       - run: sh .circleci/status.sh
-      - run:
-          name: Run specs
-          command: |
-            cd priv/cabbage
-            mix test
+      # - run:
```

Comment:
I commented cabbage test for faster load test runs. I'll uncomment them

Reply:
what if we separate cabbage and load test runs into two jobs?

Reply:
I created a separate ci step for load tests